pkg/server/backgroundsvcs/background_services.go

```diff
@@ -51,7 +52,7 @@ func ProvideBackgroundServiceRegistry(
 	_ *influxdb.Service, _ *loki.Service, _ *opentsdb.Service, _ *prometheus.Service, _ *tempo.Service,
 	_ *testdatasource.TestDataPlugin, _ *plugindashboards.Service, _ *dashboardsnapshots.Service, _ secrets.Service,
 	_ *postgres.Service, _ *mysql.Service, _ *mssql.Service, _ *grafanads.Service, _ *cloudmonitoring.Service,
-	_ *pluginsettings.Service, _ *alerting.AlertNotificationService,
+	_ *pluginsettings.Service, _ *alerting.AlertNotificationService, _ serviceaccounts.Service,
```

Comment:
registering serviceaccounts as a background service

Reply:
Why are service accounts a background service?

Reply:
---
it seems like background services hold a handler (aka, shared state).
service accounts does not, it only provides the api endpoints. therefore this should probably not be a background service.
@sakjur 

option1: we could just reigster the api endpoints and build the struct once it is being called
option2: register as a service on the http server